src/content/getting-started/quickstart/external/_index.md

```diff
@@ -0,0 +1,429 @@
+---
+date: 2021-08-11T12:33:18+01:00
+title: "External Network (Experimental)"
+weight: 15
+---
+
+This guide covers the minimum steps to try external network use case.
+In this use case, users can access between pods in clusters and applications outside clusters by using DNS names that lighthouse provides or globalnet ingress IPs.
+In addition, source IPs of these communications can be preserved.
+
+This feature is still experimental, so the behaviors and configurations may be changed in the future.
+
+### Prerequisites
+
+1. Prepare below:
+    - Two or more k8s clusters
+    - One or more non-cluster hosts, that exist in the same network segment to the one of the k8s clusters
+
+    In this guide, we will use the following k8s clusters and a non-cluster host.
+
+    - K8s clusters:
+
+    | Cluster   | Node IP(s)     | Remarks             |
+    |-----------|----------------|---------------------|
+    | cluster-a | 192.168.122.26 | Single-node cluster |
+    | cluster-b | 192.168.122.27 | Single-node cluster |
+
+    - non-cluster host:
+
+    | Host      | IP              | Remarks            |
+    |-----------|-----------------|--------------------|
+    | test-vm   | 192.168.122.142 | Linux is installed |
+
+    In above case, all of them are deployed inside 192.168.122.0/24 segment.
+    However, it is only required that cluster-a and test-vm are in the same segment.
+    Other clusters, cluster-b and any additional clusters, can be deployed in different segments or even in any other networks in the internet. Also, these clusters can be multi-node cluster.
+
+    On the other hand, subnets of non-cluster hosts should be distinguished from those of all the clusters to easily specify the external network CIDR.
+    In above case, cluster-a and cluster-b belongs to 192.168.122.0/25 network and test-vm belongs to 192.168.122.128/25 network.
+    Therefore, the external network CIDR for this configuration is 192.168.122.128/25.
+    In test environment for just one host, we will be able to specify external network CIDR, like 192.168.122.142/32, however design of the subnet needs to be considered when more hosts are added.
```

Comment:
For this configuration to be more flexible, it would be better to provide a separate interface for external CIDR, in the future, as @mangelajo has already pointed out.